### PR TITLE
BinaryProvider for Julia 1.1, 1.2

### PR DESCRIPTION
**Pull request by wookay**

### CODE REVIEW

File: src/FIGlet.jl [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-using Pkg.Artifacts
-import Base
+using Pkg
 
-const FONTSDIR = abspath(normpath(joinpath(artifact"fonts", "FIGletFonts-0.5.0", "fonts")))
+if isdefined(Pkg, :Artifacts)
+    using Pkg.Artifacts
+    @eval fontsroot = artifact"fonts"
+else
+    fontsroot = normpath(@__DIR__, "..", "deps")
```
Comment (kdheepak) [inline src/FIGlet.jl]: Can you make this `FONTSDIR` instead of `fontsroot`?

**Comment by kdheepak** [comment]
Thanks for the PR. Do you know why the tests don't pass?

**Comment by wookay** [comment]
nice!

**Comment by kdheepak** [comment]
Thanks for making the PR!

**Comment by wookay** [comment]
you need `deps/build.jl` file on master branch.

* https://github.com/kdheepak/FIGlet.jl/blob/kd/julia-binary-provider-support/deps/build.jl
* https://github.com/kdheepak/FIGlet.jl/tree/master

**Comment by wookay** [comment]
oh. never mind.

**Comment by kdheepak** [comment]
I'm planning to maintain the version for Julia 1.0, 1.1 and 1.2 in a separate branch, as an experiment. I think once people adopt 1.3 I'll up end up removing the `deps/build.jl` and `BinaryProvider` related code, so might as well leave `master` as is.

**Pull request merged**